admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/SecurityConfig.kt

```diff
@@ -118,39 +153,43 @@ class SecurityConfig(override val di: DI) : DIAware {
 }
 
 fun Routing.oauthRoutes() {
+    val oauthConfig by di().instance<OAuthConfig>()
+
     authenticate("oauth") {
         get("/oauth/login") {
-            // Redirects to "authorizeUrl" if authentication fails
-            // or redirect to root otherwise
-            call.respondRedirect("/")
-        }
-        get("/oauth/logout") {
-            call.principal<OAuthAccessTokenResponse.OAuth2>()?.let { principal ->
-                val httpClient = HttpClient(Apache)
-                httpClient.post<HttpResponse>("http://localhost:8080/realms/master/protocol/openid-connect/logout") {
-                    headers {
-                        append(HttpHeaders.Authorization, "Bearer ${principal.accessToken}")
-                    }
-                }
-            }
+            // Redirects to "authorizeUrl" automatically
```

Comment:
Will it work without specifying the `get("/oauth/login")` route? (empty handler looks odd)

Reply:
no we can't, it'll 404